frontend/src/routes/train.jsx

```diff
@@ -0,0 +1,120 @@
+import { styled } from "solid-styled-components";
+import { createFormGroup, createFormControl } from "solid-forms";
+import Button from "../components/form/button";
+import EmailInput from "../components/form/email-input";
+import HashInput from "../components/form/hash-input";
+import FileInput from "../components/form/file-input";
+import Slider from "../components/slider";
+import { getFilesUrls, validateEmail } from "../helpers";
+import { postTrain } from "../api";
+import { config } from "../config";
+
+const Train = () => {
+  const group = createFormGroup({
+    email: createFormControl("", {
+      validators: (value) =>
+        validateEmail(value)
+          ? null
+          : { errorMessage: "Email address is invalid" },
+    }),
+    files: createFormControl([], {
+      validators: (files) =>
+        files.length < config.minFiles
+          ? {
+              errorMessage: `There must be ${config.minFiles} images or more to start training`,
+            }
+          : null,
+    }),
+    hash: createFormControl("", { readonly: true, disabled: true }),
+  });
+
+  const imageList = () => getFilesUrls(group.controls.files.value);
+
+  const handleFormSubmit = async () => {
+    if (group.isSubmitted) {
+      return;
+    }
+    if (!group.controls.email.isValid) {
+      group.controls.email.markTouched(true);
+    } else if (!group.controls.files.isValid) {
+      group.controls.files.markTouched(true);
+    } else {
+      // temp const
+      const isMock = true;
```

Comment:
You can add this constant to the config and use it in all cases where we need mocks now